### PR TITLE
ansible 1.9.0.1

### PR DESCRIPTION
**Pull request by tnayuki**

### CODE REVIEW

File: Library/Formula/ansible.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Ansible < Formula
   homepage "http://www.ansible.com/home"
-  url "http://releases.ansible.com/ansible/ansible-1.8.4.tar.gz"
-  sha1 "74be0c102506d14940bc76f5c5659618368c41ec"
+  url "http://releases.ansible.com/ansible/ansible-1.9.0.1.tar.gz"
+  sha1 "a61c0dfdb3f395546310999d2b75c2ce422fc1ed"
   revision 1
```
Comment (mikemcquaid) [inline Library/Formula/ansible.rb]: Please remove the `revision`, thanks!

**Comment by tnayuki** [comment]
Is this okay?

**Comment by mikemcquaid** [comment]
Should be, thanks!

**Comment by tdsmith** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Comment by xu-cheng** [comment]
It seems that this PR has been merged in 617e0fe256d892ad152ca3d51d542e087743add9 with closing it. Close it manually.

**Pull request closed without merging**